plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraTableHandle.java

```diff
@@ -14,79 +14,107 @@
 package io.trino.plugin.cassandra;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.google.common.collect.ImmutableList;
 import io.trino.spi.connector.ConnectorTableHandle;
 import io.trino.spi.connector.SchemaTableName;
 
 import java.util.List;
 import java.util.Objects;
 import java.util.Optional;
-import java.util.stream.Stream;
 
-import static java.lang.String.format;
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static com.google.common.base.Preconditions.checkState;
 import static java.util.Objects.requireNonNull;
-import static java.util.stream.Collectors.joining;
 
 public class CassandraTableHandle
         implements ConnectorTableHandle
 {
-    private final String schemaName;
-    private final String tableName;
-    private final Optional<List<CassandraPartition>> partitions;
-    private final String clusteringKeyPredicates;
+    private final CassandraRelationHandle relationHandle;
 
+    @Deprecated
```

Comment:
Instead of marking it as deprecated - can we remove the usage of this constructor. I think it is being used in 4 places. Maybe it could go as a preparatory commit or a different PR in the followup

Reply:
Can the changes related to removal of the Deprecated constructor - be a different commit.